devel/kchmviewer/files/patch-use-webkit-and-dbus.diff

```diff
@@ -0,0 +1,71 @@
+diff --git a/src/src.pro b/src/src.pro
+index 2524b5e..f9e7098 100644
+--- a/src/src.pro
++++ b/src/src.pro
+@@ -40,7 +40,7 @@ SOURCES += config.cpp \
+     textencodings.cpp \
+     treeitem_toc.cpp \
+     treeitem_index.cpp
+-LIBS += -lchm -lzip
++#LIBS += -lchm -lzip
+ TARGET = ../bin/kchmviewer
+ CONFIG += threads \
+     warn_on \
+@@ -81,7 +81,7 @@ macx: {
+     HEADERS += kchmviewerapp.h
+     SOURCES += kchmviewerapp.cpp
+     QMAKE_INFO_PLIST=resources/Info.plist
+-    QMAKE_POST_LINK += cp resources/*.icns ${DESTDIR}/kchmviewer.app/Contents/Resources;
++    ICON = resources/kchmviewer.icns
+     LIBS += ../lib/libebook/libebook.a
+     POST_TARGETDEPS += ../lib/libebook/libebook.a
+ }
+@@ -102,7 +102,7 @@ win32-*: {
+     LIBS += -lwsock32 -loleaut32
+ }
+ 
+-unix:!macx: {
++unix: {
+ 
+     QT += dbus
+     HEADERS += dbus_interface.h
+@@ -112,6 +111,9 @@ unix: {
+     POST_TARGETDEPS += ../lib/libebook/libebook.a
+ }
+ 
++LIBS += -L@PREFIX@/lib -lchm -L@PREFIX@/lib/ -lzip
```

Comment:
Maybe the fact I left the previous location as a comment puts people on the wrong foot? The point here is to move the place where the libraries are added on the commandline, i.e. at the end. You'd say that this shouldn't matter, but that's not what I experienced across the different systems where I tested the build. That's also why there are 2 -L options, with a slightly different argument. Apparently the options applies fully only to the 1st following `-l`; without the 2nd -L I was getting the wrong `-lzip`. The slightly different location (trailing /) is to prevent the linker from skipping the 2nd -L when it's exactly the same.